src/app/home/home.component.html

```diff
@@ -42,7 +42,7 @@ <h2>Grassroots Spirit</h2>
 </main>
 <footer>
     <span class="footerLogo">
-        <img src="../../assets/CashmereLogoWhite.svg" height="40px" width="auto" style="fill: #ffffff;" alt="Fabric.Cashmere Logo"/>
+        <img src="../../assets/CashmereLogoWhite.svg" height="40px" width="auto" alt="Fabric.Cashmere Logo"/>
     </span>
     <span class="footerVersion">
         <p>Current Version: <b>0.1 (pre-release)</b></p>
```

Comment:
according to npmjs.org the current version is `0.0.2` https://www.npmjs.com/package/@healthcatalyst/cashmere

Reply:
Hah, okay I just made that number up, so that's great to know.